pkg/agent/flowexporter/types.go

```diff
@@ -51,19 +52,27 @@ type Connection struct {
 	Labels, LabelsMask []byte
 	// TODO: Have a separate field for protocol. No need to keep it in Tuple.
 	TupleOrig, TupleReply          Tuple
+	FlowKey                        Tuple
```

Comment:
For deny connections, we don't have `TupleReply` field, so I added this to avoid confusion. And it's easier to get ConnKey out of this FlowKey. Do you mean we can assign 5-Tuple info of connection to `TupleOrig` and leave `TupleReply` empty for deny connections?

Reply:
Yes reusing the field is better rather than increasing the size of the struct.
In addition, we could merge `TupleOriginal` and `TupleReply by maintaining SNAT IP/Port and DNAT IP/Port (we already have cluster IP field) separately. We can make the struct leaner for conntrack connections too.